README.rst

```diff
@@ -106,6 +106,12 @@ compose. For this you need to have the following installed.
 Use ``sudo docker-compose up`` to start VulnerableCode. Then access
 VulnerableCode at http://localhost:8000/ or at http://127.0.0.1:8000/
 
+If you run into a ``ModuleNotFoundError: No module named '<module>'``, run::
```

Comment:
let's make this more general. 

The root cause of problem here is that after upgrading the code, the docker setup is not synced ie new migrations, deps are not applied. 

So a better solution to fix all such problems would be to do 
`sudo docker-compose up -d --no-deps --build web`

So I would change this to something like 
```
Don't forget to run `sudo docker-compose up -d --no-deps --build web` to sync your instance after every `git pull`.
```

Reply:
oh, great. Running this will sync the docker environment each time the codebase updates